ql/src/test/resources/org/apache/hadoop/hive/ql/udf/generic/TestGenericUDFDateFormatEvaluate.csv

```diff
@@ -15,7 +15,7 @@ Input datetime;Input pattern;Local timezone;Formatter type;Expected output
 1800-01-01 00:00:00;yyyy-MM-dd HH:mm:ss;Asia/Kolkata;SIMPLE;1799-12-31 23:36:32
 Jul 9 2023;MMM dd yyyy;Etc/GMT;DATETIME;null
 Jul 9 2023;MMM dd yyyy;Etc/GMT;SIMPLE;null
-2023-07-21;DD;Etc/GMT;DATETIME;null
+2023-07-21;DD;Etc/GMT;DATETIME;202
```

Comment:
Why?

Reply:
This is due to change in behaviour of java code with respect to DAY_OF_YEAR. The detailed description is added in this ticket.
https://issues.apache.org/jira/browse/HIVE-28381